README.md

```diff
@@ -1,28 +1,62 @@
-# TPS Report
+# "Third Party Software" Report
+
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
+1. [Deployment](#deployment)
+1. [Contributing](#contributing)
+
+## About the Project
+
+This started as a Passion Project idea from [@jadametz](https://github.com/jadametz) to solve a pain point in how teams stay on top of new releases
+for all the software they may have in use.
```

Comment:
While I also appreciate shout-outs, I think we can remove the origin story part. Just explaining what the project is is what's important.